src/plugins/console/public/lib/autocomplete/body_completer.js

```diff
@@ -204,7 +204,9 @@ function compileParametrizedValue(value, compilingContext, template) {
   value = value.substr(1, value.length - 2).toLowerCase();
   let component = compilingContext.parametrizedComponentFactories.getComponent(value, true);
   if (!component) {
-    throw new Error("no factory found for '" + value + "'");
+    console.warn("[Console] no factory found for '" + value + "'");
+    // using upper case as an indication that this value is a parameter
+    return new ConstantComponent(value.toUpperCase());
```

Comment:
Instead of throwing an error and failing all definitions loading, we log a warning and use a constant with an upper case string.